lib/zeus/api_client/trigalerts_interface.rb

```diff
@@ -0,0 +1,32 @@
+require 'zeus/api_client/rest_interface'
+require 'zeus/api_client/result'
+
+module Zeus
+  module TrigalertsInterface
+    include RestInterface
+
+    # get triggered alerts
+    # @return [Zeus::APIClient::Result]
+
+    def get_triggered_alerts()
+      begin
+        response = get("/trigalerts/#{@access_token}")
+        Result.new(response)
+      rescue => e
+        Result.new(e.response)
+      end
+    end
+
+    # get triggered alerts in last 24 hours
+    # @return [Zeus::APIClient::Result]
+
+    def get_triggered_alerts_last_24_hours()
```

Comment:
Actually, `begin` statement is redundant for functions.
So we can write like following (actually this is my fault):

```ruby
    def get_triggered_alerts_last_24_hours
      response = get("/trigalerts/#{@access_token}/last24")
      Result.new(response)
    rescue => e
      Result.new(e.response)
    end
```

We dont need to re-write like this, but just I let you know ;)